lib/fingerprint/scanner.rb

```diff
@@ -96,15 +98,21 @@ def digests_for(path)
 					@progress.call(total) if @progress
 					
 					@digests.each do |key, digest|
-						digest << buf
+						digest << buf unless key == 'BLAKE2s'
```

Comment:
I am okay to merge this, but I'd like to add a layer of abstraction here to reduce the special casing.